### PR TITLE
Feature/print behavior

### PR DESCRIPTION
**Pull request by Alexander-Makaryev**
implementation of https://github.com/IntelPython/daal4py/issues/7
this is continuation of https://github.com/IntelPython/daal4py/pull/45 but into https://github.com/IntelPython/daal4py/tree/2019.3 branch

### CODE REVIEW

**Comment by fschlimb** [comment]
@oleksandr-pavlyk do you agree with the fix?
@triskadecaepyon As the requester of this feature pls comment on the output as mentioned in #45

File: generator/wrapper_gen.py [hunk 1]
```diff
@@ -189,7 +189,7 @@ def _str(instance, properties):
                 result += p + ': dict, len={}'.format(len(prop))
             else:
                 result += p + ': '
-            value = str(prop).replace('\\n', '\\n  ')
+            value = '\\n  '.join(str(prop).splitlines())
```
Comment (fschlimb) [inline generator/wrapper_gen.py]: why do you use ```\\n``` and not simple ```\n```?

Comment (oleksandr-pavlyk) [reply generator/wrapper_gen.py]: Please replace all explicit `'\\n'` with use of `os.linesep` variable for portability.

**Comment by oleksandr-pavlyk** [comment]
@Alexander-Makaryev This printing behavior can be improved. Consider

```python
import daal4py
import numpy as np
np.random.seed(0)
X = np.random.randn(20,3)
res = daal4py.low_order_moments(fptype='double').compute(X)
print(res)
```

It currently gives

```
In [15]: print(res)
minimum: array, type=float64, shape=(1, 3)[[-1.98079647 -1.45436567 -2.55298982]]

maximum: array, type=float64, shape=(1, 3)[[2.26975462 1.9507754  1.46935877]]

sum: array, type=float64, shape=(1, 3)[[ 5.5146903   3.9174313  -4.83208387]]

sumSquares: array, type=float64, shape=(1, 3)[[27.10978743 20.95621865 19.68242165]]

sumSquaresCentered: array, type=float64, shape=(1, 3)[[25.58919697 20.18890525 18.51496992]]

mean: array, type=float64, shape=(1, 3)[[ 0.27573452  0.19587156 -0.24160419]]

secondOrderRawMoment: array, type=float64, shape=(1, 3)[[1.35548937 1.04781093 0.98412108]]

variance: array, type=float64, shape=(1, 3)[[1.34679984 1.06257396 0.9744721 ]]

standardDeviation: array, type=float64, shape=(1, 3)[[1.16051706 1.03081228 0.98715353]]

variation: array, type=float64, shape=(1, 3)[[ 4.20882042  5.26269488 -4.08582947]]
``

Clarity of presentation would improve if the type and dtype were deferred to printing of `ndarray` itself.
For example:

```
In [19]: repr(res.minimum)
Out[19]: 'array([[-1.98079647, -1.45436567, -2.55298982]])'

In [20]: repr(res.minimum.astype('f'))
Out[20]: 'array([[-1.9807965, -1.4543657, -2.5529897]], dtype=float32)'
```

The shape is a redundant information, which is self-evident from the array itself. 

For low order moments, printing should provide _useful_ summary. Duplication of `mean` and `sum`, `sumSquared` and `secondOrderRawMoment` are redundant, but without writing class specific special code this might have to do, though.

**Comment by fschlimb** [comment]
@oleksandr-pavlyk the shape is only evident if the array is small enough. For larger arrays we only see the beginning and end. Maybe we should print the shape only if we do not print the entire array?

**Comment by fschlimb** [comment]
We could also have the values of arrays in a separate line - always.

**Comment by Alexander-Makaryev** [comment]
New version of printing examples:
```
minimum: array(
  [[-9.948613 -9.737347 -9.619284 -9.743521 -9.990568 -9.9973   -9.997706
    -9.897363 -9.794658 -9.861839]],
  dtype=float64, shape=(1, 10))

maximum: array(
  [[9.90086  9.869278 9.997498 9.868528 9.404993 9.768626 9.910092 9.896053
    9.647098 9.971762]],
  dtype=float64, shape=(1, 10))

sum: array(
  [[ -13.894732   65.933167   30.511676   40.409349  -64.703292  -94.211596
      32.531087   69.566359 -131.328752   42.703151]],
  dtype=float64, shape=(1, 10))

sumSquares: array(
  [[7249.157216   6569.17370838 6068.50429546 5977.01040518 6720.27291173
    6209.56635712 6550.82550358 6637.25266512 6995.62512713 7102.79385404]],
  dtype=float64, shape=(1, 10))

sumSquaresCentered: array(
  [[7248.19189812 6547.43779583 6063.8494836  5968.84582775 6699.34033176
    6165.18723302 6545.53414547 6613.05527359 6909.38892162 7093.67605851]],
  dtype=float64, shape=(1, 10))

mean: array(
  [[-0.06947366  0.32966584  0.15255838  0.20204675 -0.32351646 -0.47105798
     0.16265544  0.34783179 -0.65664376  0.21351576]],
  dtype=float64, shape=(1, 10))

secondOrderRawMoment: array(
  [[36.24578608 32.84586854 30.34252148 29.88505203 33.60136456 31.04783179
    32.75412752 33.18626333 34.97812564 35.51396927]],
  dtype=float64, shape=(1, 10))

variance: array(
  [[36.42307486 32.90169747 30.47160545 29.99420014 33.66502679 30.98084037
    32.89213138 33.23143354 34.72054734 35.64661336]],
  dtype=float64, shape=(1, 10))

standardDeviation: array(
  [[6.03515326 5.73600013 5.52010919 5.4766961  5.80215708 5.56604351
    5.7351662  5.76467116 5.89241439 5.97047849]],
  dtype=float64, shape=(1, 10))

variation: array(
  [[-86.86966052  17.39943761  36.18358551  27.10608427 -17.93465804
    -11.81604759  35.25960386  16.5731576   -8.97353291  27.96270695]],
  dtype=float64, shape=(1, 10))
```
```
model:
  NumberOfBetas: 11

  NumberOfResponses: 2

  InterceptFlag: True

  Beta: array(
    [[ 2.41272553  9.25181248  0.3856898  -6.28711921 -4.12093117  7.77676348
       7.83651727  3.30835195  9.10860594 -6.73231343 -1.83129615]
     [ 2.47020425 -4.97377057  1.17108509  8.94688555 -7.54706096  2.71588871
       6.26894149  7.72190342 -8.53511092  1.71497193  5.05079029]],
    dtype=float64, shape=(2, 11))

  NumberOfFeatures: 10
```

**Comment by Alexander-Makaryev** [comment]
Example of one-line array output in printing of decision forest classification training results:
```
model:
  NumberOfTrees: 10

  NFeatures: 3

  NumberOfFeatures: 3

outOfBagError: array(
  [[0.03298523]],
  dtype=float64, shape=(1, 1))

variableImportance: array(
  [[78.01831326 79.08990148  0.28486318]],
  dtype=float64, shape=(1, 3))

outOfBagErrorPerObservation: None
```

**Comment by oleksandr-pavlyk** [comment]
@Alexander-Makaryev Much cleaner. LGTM!

**Pull request merged**